packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -111,9 +108,8 @@ library LibVerifying {
 
             if (fc.prover == address(0)) break;
 
-            uint256 proofCooldownPeriod = oracleProver == address(0) ||
-                fc.prover == oracleProver
-                ? 0
+            uint256 proofCooldownPeriod = fc.prover == address(1)
```

Comment:
Rewards will get sent to `address(1)`, may be problematic if we want to give these tokens value?

Reply:
Well, yes we can / should figure out something. Currently rewards (for the system prover) will be held in the protocol (`state.taikoTokenBalances[address(1)]`).
Some notes:
- It will have a 15 min cool down period, so people might overwrite this block so they will be beneficiaries.
- We can add a specific role/withdraw function to claim this reward on behalf of X.
- Or we can even set a specific address at reward amount accounting if `address(1)`

Let me know which one would be best.

Reply:
Last option seems simplest to me, but no strong opinion.

Reply:
If prover == address(1), simple send to "system_prover".

Reply:
Fixed